backend/tests/email.rs

```diff
@@ -0,0 +1,65 @@
+use actix_web::web::Data;
+use hack4krak_backend::utils::app_state::AppState;
+use hack4krak_backend::utils::emails::{Email, EmailTemplate};
+use lettre::transport::smtp::client::Tls;
+use lettre::SmtpTransport;
+use serde_json::Value;
+use testcontainers::core::{IntoContainerPort, WaitFor};
+use testcontainers::runners::AsyncRunner;
+use testcontainers::GenericImage;
+
+#[actix_web::test]
+async fn send_mail() {
```

Comment:
Hmm... Maybe make tests using testcontainers opt-in? Some users might want to run other tests without running those.